src/msg/async/Event.cc

```diff
@@ -126,10 +126,8 @@ EventCenter::~EventCenter()
   }
   assert(time_events.empty());
 
-  if (notify_receive_fd >= 0) {
-    delete_file_event(notify_receive_fd, EVENT_READABLE);
+  if (notify_receive_fd >= 0)
```

Comment:
will what? i guess you mean "nuke" or something? and what does "in the following" stands for? seems this commit does not look right by its own, could you move it to the commit where "we will the nuke whole epoll"?